Documentation/dev/developer-guide.md

```diff
@@ -136,6 +136,11 @@ When committing new dependencies, please use the following guidelines:
   - `Gopkg.toml: Add new dependency $your_new_dependency`
   - `Gopkg.lock,vendor: Add new dependency $your_new_dependency`
 
+## Helm templates
+
+If you have added a new Helm Chart and would like to render the template to check the values and nesting within the yaml file you can run:
+helm template CHART_DIR -x CHART.yaml
```

Comment:
A few things:
- Commands should be wrapped in triple backticks to put it into a code block
- The 2nd argument is the template path, this isn't as clear as it could be. Maybe `-x path/to/template/file.yaml`